Testscripts/Linux/KVP-MaxRead.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+# Source utils.sh
+. utils.sh || {
+    echo "ERROR: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 0
+}
+UtilsInit
+
+
+if [ "$(ps aux | grep -c [k]vp)" -lt 1 ]; then
+    LogErr "KVP daemon is not running"
+    SetTestStateAborted
+    exit 0
+fi
+
+# Verify OS architecture, select kvp client
+if [ "$(uname -a | grep -c x86_64)" -eq 1 ]; then
+    LogMsg "64 bit architecture was detected"
+    kvp_client="kvp_client64"
+elif [ "$(uname -a | grep -c i686)" -eq 1 ]; then
+    LogMsg "32 bit architecture was detected"
+    kvp_client="kvp_client32"
+else
+    LogErr "Unable to detect OS architecture"
+    SetTestStateAborted
+    exit 0
+fi
+
+# Append stage 1
+value="value"
+key="test"
+counter=0
+while [ "$counter" -le "$Entries" ]; do
```

Comment:
Add log msg about what it is trying to do.

Reply:
added log message after each append